src/ZODB/FileStorage/FileStorage.py

```diff
@@ -536,9 +535,7 @@ def store(self, oid, oldserial, data, version, transaction):
                     "The storage quota has been exceeded.")
 
             if old and oldserial != committed_tid:
-                return ResolvedSerial
-            else:
-                return self._tid
+                self._resolved.append(oid)
```

Comment:
Move this line after the the call to ``self.tryToResolveConflict`` and get rid of this ``if``.